### PR TITLE
[ONNX] Fix case for `activations` attribute in nn.RNN ONNX export.

### PR DESCRIPTION
**Pull request by spandantiwari**
This PR addresses the https://github.com/pytorch/pytorch/issues/19366 issue.

### CODE REVIEW

File: torch/onnx/symbolic.py [hunk 1]
```diff
@@ -1454,6 +1454,9 @@ def group_norm(g, input, num_groups, weight, bias, eps, cudnn_enabled):
 
 def _generic_rnn(g, variant, input, initial_states, all_weights, has_biases,
                  num_layers, dropout, train, bidirectional, batch_first=None, batch_sizes=None):
+    onnxActivations = ['Relu', 'Tanh', 'Sigmoid', 'Affine', 'LeakyRelu', 'ThresholdedRelu',
+                       'ScaledTanh', 'HardSigmoid', 'Elu', 'Softsign', 'Softplus']
+    variantToOnnxActivationMap = dict(zip([act_fun.lower() for act_fun in onnxActivations], onnxActivations))
```
Comment (VitalyFedyunin) [inline torch/onnx/symbolic.py]: No need to zip
```
variantToOnnxActivationMap = { (act_fun.lower(), act_fun) for act_fun in onnxActivations  } 
```

**Review by VitalyFedyunin — DISMISSED** [review]

**Review by houseroad — COMMENTED** [review]
Seems lower/upper case mismatching problem: https://github.com/pytorch/pytorch/blob/0d663cec3068f8d22064fb5d691a1b90775e42ea/caffe2/python/rnn_cell.py#L314

File: torch/onnx/symbolic.py (hunk 1 above)
Comment (spandantiwari) [reply torch/onnx/symbolic.py]: Without zip it creates a set, and I meant to create a dict.

**Comment by spandantiwari** [comment]
> Seems lower/upper case mismatching problem:
>
>  if self.activation not in ['relu', 'tanh']:

Thanks @houseroad . This was the issue. Submitted an update.

**Comment by spandantiwari** [comment]
@houseroad - checking if this is ready for merge.

**Review by houseroad — APPROVED** [review]
@spandantiwari thanks for fixing the issue :-)

**Comment by spandantiwari** [comment]
Thanks @houseroad

**Pull request closed without merging**